Sources/ImageDownloader.swift

```diff
@@ -141,6 +153,9 @@ extension ImageDownloaderDelegate {
     public func isValidStatusCode(_ code: Int, for downloader: ImageDownloader) -> Bool {
         return (200..<400).contains(code)
     }
+    public func imageDownloader(_ downloader: ImageDownloader, didDownload data: Data, for url: URL) -> Data? {
+        return nil
```

Comment:
Maybe we should just return the original data instead of `nil`? Any reason to return a `nil` here?

Reply:
You're right, there's no particular reason, to return nil. I've pushed updated version.